themes/ranked-choice-voting-for-texas-5dabb2094445ea66232bca7e/_flash_share.html

```diff
@@ -1,4 +1,4 @@
-<div id="flash-share" class="flash-share form-wrap">
+<div id="flash-share" class="flash-share form-wrap" style="display:none;">
```

Comment:
Is this and the other `display:none` in _nav.html set up to work with javascript?  I'm assuming they are, but just want to make sure they shouldn't just be deleted.

Reply:
yes